### PR TITLE
Some bug fixes for potentially undefined keys

### PR DESCRIPTION
**Pull request by wmtech-1**

### CODE REVIEW

**Comment by wmtech-1** [comment]
I don't like that either. Just tried to follow the coding style as seen at Facebook.php. ;)

File: src/Backend/Reddit.php [hunk 1]
```diff
@@ -34,8 +34,10 @@ public function getRequest($url)
     public function extractCount(array $data)
     {
         $count = 0;
-        foreach ($data['data']['children'] as $child) {
-            $count += $child['data']['score'];
+        if (!empty($data['data']['children']) && !empty($child['data']['score'])) {
```
Comment (liayn) [inline src/Backend/Reddit.php]: `$child` is not defined here, your IDE should have shouted out loud.

**Comment by liayn** [comment]
I would suggest to squash all those commits together into a single cleanup commit

**Comment by liayn** [comment]
Pushed the squashed commits as: https://github.com/heiseonline/shariff-backend-php/pull/67

**Comment by compeak** [comment]
Closed in favour of #67.

**Pull request closed without merging**